modules/phenyl-api-explorer/src/client/src/components/OperationEditor/index.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react'
+import { Form } from 'semantic-ui-react'
+
+type Props = {
+  match: any,
+  sessionId: string,
+  operations: Array<string>,
+  defaultPayloads: { [string]: Object },
+  execute: ({ entitiName: string, operation: string, payload: any }) => any,
+}
+
+type State = {
+  operation: string, // FIXME: enum
+  payload: Object,
+}
+
+class OperationEditor extends Component<Props, State> {
+  state: State = {
+    operation: null,
+    payload: null,
+  }
+
+  handleChangeOperation = (event, { value }) => {
+    if (!this.props.defaultPayloads[value]) {
+      throw new Error(`Unknown operation: ${value}`)
+    }
+
+    const payload = this.props.defaultPayloads[value]
+
+    this.setState({
+      operation: value,
+      payload: JSON.stringify(payload, null, 2),
+    })
+  }
+
+  handleChangePayload = (event, { value }) => {
+    this.setState({ payload: value })
+  }
+
+  handleRun = () => {
+    const { match, execute } = this.props
+
+    const payload = JSON.parse(this.state.payload)
+    execute({
+      sessionId: this.props.sessionId,
+      entityName: match.params.functional,
+      operation: this.state.operation,
```

Comment:
I prefer `method` over `operation`, as operation has different meanings in phenyl world.
Would you mind if you change the name?